examples/bf-run

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
+
+set -e
+
+# Start Buildfarm Cluster
+start_buildfarm () {
+  # Run Redis Container
+  docker run -d --name buildfarm-redis -p 6379:6379 redis:5.0.9
+
+  # Run Buildfarm Server Container
+  docker run -d --name buildfarm-server -v $(pwd)/examples:/var/lib/buildfarm-server -p 8980:8980 --network host \
+  80dw/buildfarm-server:latest /var/lib/buildfarm-server/shard-server.config.example -p 8980
+
+  # Run Buildfarm Shard Worker Container
```

Comment:
Out of curiosity, how come the buildfarm worker must run in a privileged container? And why does it need `host` networking?

Reply:
Privileged container is required if you are utilizing exec_properties. Otherwise it can be left off. Host networking is used to be able to reference locally running redis by localhost as in the example configs. If you specify a redis instance as an actual IP address this can also be left off. The idea here was to stand up a fully functioning Buildfarm cluster with the least amount of effort possible.

Reply:
Awesome. Thanks for explaining! :)